docs/BacwardCompatibility.md

```diff
@@ -0,0 +1,224 @@
+[pkg/model](https://github.com/fybrik/fybrik/tree/master/pkg/model), therefore any changes in the directory should be  
+validated for backward compatibility.
+- The basic [taxonomy file](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/taxonomy.json), is
+autogenerated from the both above sources. However, users are able to overwrite it, as describe by [Using a Custom 
+Taxonomy for Resource Validation](https://fybrik.io/v1.1/tasks/custom-taxonomy/). 
+
+Unfortunately all these files do not have version definition and it can be complicated to separate files from different 
+releases. In order to be able to compare the files, I suggest to add an entry `version` and modify the 
+[json-schema-generator](https://github.com/fybrik/json-schema-generator) project to generate the entry. It's an optional 
+entry, so we don't have to change the go structures, but will be able to validate the content of the 
+`fybrik-taxonomy-config` configuration map. Due to possibility of user modifications of the basic taxonomy file I suggest 
+in addition to the version, add the data of the file generation and maybe its CRC. 
+
+*Note:* Should we create a taxonomy json file for the Credential Management connector?
+*Note:* Should we provide a [default metric taxonomy](https://fybrik.io/v1.1/tasks/infrastructure/#add-a-new-attribute-definition-to-the-taxonomy)
+
+## Default Policies
+Fybrik is installed with a set of different default policies. Most of the change in the policies do not prevent Fybrik 
+operation, but can chnage its default behaviour and therefore influence on its users.
+
+### Default OPA Policy Rules
+The [default OPA policy rules](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/opa-server/policy-lib/default_policy.rego) 
+defines Fybrik behavior if no other rules are not provided by users.
+
+### Default IT Config Policies 
+[IT config policies](https://fybrik.io/v1.1/concepts/config-policies/) are the mechanism via which the organization may 
+influence the construction of the data plane, taking into account infrastructure capabilities and costs.
+Out of the box policies come with the fybrik deployment. They define the deployment of basic capabilities, 
+such as read, write, copy and delete.
+The default IT policies and the default infrastructure definitions locates at 
+[./charts/fybrik/adminconfig](https://github.com/fybrik/fybrik/tree/master/charts/fybrik/files/adminconfig)
+
+*Note:* in order to update the IT config policies and define infrastructure, Fybrik 
+[recommends](https://fybrik.io/v1.1/concepts/config-policies/#how-to-provide-custom-policies) to overwrite the files in 
+the `./charts/fybrik/adminconfig` or direct changes in the `fybrik-adminconfig` configmap. However, fybrik updates will 
+remove these changes. We have to provide a different mechanism, which will automatically merge default and user defined 
+settings.  
```

Comment:
Please open an issue for this

Reply:
See https://github.com/fybrik/fybrik/issues/1769